### PR TITLE
Add SCIM Support

### PR DESCRIPTION
**Pull request by ollie-stytch**

### CODE REVIEW

File: lib/b2b/index.ts [hunk 1]
```diff
@@ -81,8 +80,8 @@ export type {
 
 export type {
   B2BOrganizationsMembersOAuthProvidersGoogleResponse,
+  B2BOrganizationsMembersOAuthProvidersMicrosoftRequest,
   B2BOrganizationsMembersOAuthProvidersMicrosoftResponse,
-  B2BOrganizationsMembersOAuthProvidersProviderInformationRequest,
```
Comment (logan-stytch) [inline lib/b2b/index.ts:85]: I think you need to update either protobuf or sdk-codegen

Comment (logan-stytch) [reply lib/b2b/index.ts:85]: And then probably reset _this_ repo to re-add the manually added type (before regenerating)

**Review by logan-stytch — CHANGES_REQUESTED** [review]

File: lib/b2b/index.ts (hunk 1 above)
Comment (ollie-stytch) [reply lib/b2b/index.ts]: Was just going through this! Should be good now. Thx Logan!

File: lib/b2b/index.ts [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 // MANUAL(exports)(TYPES)
 export type { SearchQueryOperand } from "./organizations";
-export type { B2BOrganizationsMembersOAuthProvidersMicrosoftRequest } from "./organizations_members_oauth_providers";
```
Comment (logan-stytch) [inline lib/b2b/index.ts:9]: Add this one back again -- it got removed after the first codegen, so you'll need to reset the repo or just add it back in manually once

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "stytch",
-  "version": "10.11.0",
+  "version": "10.13.0",
```
Comment (logan-stytch) [inline package.json:3]: Wait lol what? Should be 10.12. Must've accidentally done `npm version minor` twice, you can just manually edit package.json and package-lock.json if needed (I think @jennifer-stytch dealt with this a couple weeks ago)

**Review by logan-stytch — APPROVED** [review]
Two comments to address, but approving to unblock.

**Pull request merged**